webpack.config.js

```diff
@@ -0,0 +1,69 @@
+var webpack = require('webpack');
+var path = require('path');
+var fs = require('fs');
+
+var externalModules = [
+	'mkfifo',
+	'sqlite3',
+	'mysql2',
+	'pg',
+	'mariasql',
+	'mssql',
+	'mysql',
+	'strong-oracle',
+	'oracle',
+	'oracledb',
+	'pg-query-stream'
+]
+
+// We iterate over the node modules and mark all optional dependencies as external
```

Comment:
This doesn't handle the cases where dependency `a` marks `c` as optional but `b` marks it as required